modules/telco-upgrade-upgrading-all-the-olm-operators.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-upgrading-all-the-olm-operators_{context}"]
+= Upgrading all the OLM Operators
+
+In the second phase of a multi-version upgrade, you need to approve all of the Operators and additionally add install plans for any other Operators that you want to upgrade.
+
+.Procedure
+* Follow the same procedure as outlined in "Monitoring the cluster upgrade".
```

Comment:
Isn't the monitoring part just actually watching the updates happen? It's not actually updating anything, like the title here implies (and the next sentence implies)?

Reply:
Good catch! Fixing this.